packages/file-search/src/node/file-search-service-impl.spec.ts

```diff
@@ -78,4 +78,12 @@ describe('search-service', function () {
         expect(matches).to.be.not.undefined;
         expect(matches.length).to.eq(2);
     });
+
+    it('should ignore globs passed through the search options', async () => {
+        const rootUri = FileUri.create(path.resolve(__dirname, '../../test-resources/subdir1/sub2')).toString();
+
+        const matches = await service.find('foo', { rootUris: [rootUri], defaultIgnorePatterns: ['foo'] });
```

Comment:
Just add a test using an actual glob?

Reply:
Thank you for the review. i added one more test for the glob vs string.